docs/HowToGuides/GettingStarted.md

```diff
@@ -336,12 +345,10 @@ while retaining the option of building with Ninja on the command line.
 Assuming that you have already [built the toolchain via Ninja](#the-actual-build),
 several more steps are necessary to set up this environment:
 * Generate Xcode projects with `utils/build-script --xcode --swift-darwin-supported-archs "$(uname -m)"`.
-  This will first build a few LLVM files that are needed to configure the
-  projects.
```

Comment:
I think there is no harm in leaving this in. In fact I kind of like how it gives you a bit of insight into how the Xcode project is made. The first time when I read this, what I felt was "oh that's interesting to know!" instead of "is this going to take a long time?".

Reply:
Let’s leave it in then.